activities/Write.activity/js/activity.js

```diff
@@ -349,20 +350,14 @@ define([
             })
 			});
 
-		// Save as docx
+		// Save as doc for msword
 		document.getElementById(21).addEventListener('click',function(){
 			var title = document.getElementById("title").value;
-			var content = '<!DOCTYPE html>' + document.getElementById("editor").innerHTML;
-			var orientation = "portrait";
-			var converted = htmlDocx.asBlob(content, {orientation: orientation});
-			var reader = new FileReader();
-			reader.readAsDataURL(converted); 
-			reader.onloadend = function() {
-				var inputData = reader.result;
-				var mimetype = 'application/vnd.openxmlformats-officedocument.wordprocessingml.document';                
+			$("#editor").wordExport(function(inputData){
+				var mimetype = 'application/vnd.ms-word;charset=utf-8';
```

Comment:
Guess the mimetype for Word files should be `application/msword`. See my comment on https://github.com/llaske/sugarizer/pull/367